app/views/admin/bikes/edit.html.haml

```diff
@@ -1,251 +1,256 @@
+            .col-md-6
+              .form-group
+                = label :theft_description, "Theft description"
+                = s.text_area :theft_description, placeholder: "Theft Description", class: "form-control"
+          .row
+            .col-md-6
+              .form-group
+                = f.label :estimated_value, "Estimated value"
+                = s.number_field :estimated_value, class: "form-control"
+            .col-md-6
+              .form-group#stolen-check-box
+                = f.label :stolen, "Bike is stolen"
                 = f.check_box :stolen
-            #admin-recovery-fields
-              .input-group
-              .input-group
-                .control-group.large-text
-                  %label.control-label{ for: :mark_recovered_reason }
-                    How was it recovered
-                  .controls#recovery_descr
-                    = text_area_tag :mark_recovered_reason, nil, rows: 3
-              .input-group
-                .control-group
-                  %label.control-label{ for: :mark_recovered_we_helped }
-                    Did the Index help?
-                  .controls
+              #admin-recovery-fields{style: "display: none"}
+                .input-group
+                  .form-group.large-text
+                    = label :mark_recovered_reason, "How was it recovered"
+                    #recovery_descr
+                      = text_area_tag :mark_recovered_reason, nil, rows: 3
+                .input-group
+                  .form-group
+                    = label :mark_recovered_we_helped, "Did the bike index help?"
                     = check_box_tag :mark_recovered_we_helped, true
-              .input-group
-                .control-group
-                  %label.control-label{ for: :can_share_recovery }
-                    Can we share?
-                  .controls
+                .input-group
+                  .form-group
+                    = label :can_share_recovery, "Can Share"
                     = check_box_tag :can_share_recovery, true
+        - else
+          / Don't put anything
+          / Because there might be more stolen records and it breaks without
+          / an else
 
 
 
-
-  %section.full-screen-table
-    %table.table.table-striped.table-bordered
-      %thead
-        %th
+  .full-screen-table
+    %table.table.table-striped.table-bordered.table-sm
+      %thead.thead-light
+        %th.d-none.d-lg-table-cell
           ID
-        %th
+        %th.d-none.d-md-table-cell
           Created&nbsp;<small class="convertTimezone"></small>
-        %th
+        %th.d-none.d-lg-table-cell
           Creator
         %th
           Owner email
         %th
           Current?
         %th
           Claimed?
-        %th
+        %th.d-none.d-md-table-cell
           User hidden
       %tbody
         - @bike.ownerships.each do |ownership|
           %tr
-            %td
+            %td.d-none.d-lg-table-cell
```

Comment:
While I agree that the ID doesn't need to be visible on small screens - this makes it so that the link to edit the ownership isn't visible on small screens 🙃.

I also think the "created at" column is critical functionality to view.

I think you could make this be like the bikes table, where the link is in the created at and then use the less-strong right to put the ID.